tests/phpunit/README.md

```diff
@@ -0,0 +1,33 @@
+# PHPUnit Tests
+
+PHPUnit is the official testing framework chosen by the core team to test our PHP code.
+
+This documentation will assume you have a local development environment of your own choosing. For other testing workflows, such as Docker, see the [PHP: PHP Unit](https://make.wordpress.org/core/handbook/testing/automated-testing/phpunit/) handbook page.
+
+## Setup
+
+1. Check out the wordpress-develop repository
+
+       git clone git://develop.git.wordpress.org/ wordpress-develop
```

Comment:
```suggestion
1. Check out the [`wordpress-develop` repository](https://github.com/WordPress/wordpress-develop):

       git clone git@github.com:WordPress/wordpress-develop.git
```
I wonder if this should instead refer to the Github repo, as PRs there are the typical way to contribute patches and tests.